spec/ruby/core/gc/stat_spec.rb

```diff
@@ -1,19 +1,16 @@
 require_relative '../../spec_helper'
 
 describe "GC.stat" do
-  it "supports access by key" do
-    keys = [:heap_free_slots, :total_allocated_objects, :count]
-    keys.each do |key|
-      GC.stat(key).should be_kind_of(Integer)
-    end
-  end
-
   it "returns hash of values" do
     stat = GC.stat
     stat.should be_kind_of(Hash)
     stat.keys.should include(:count)
   end
 
+  it "can return a single value" do
+    GC.stat(:count).should be_kind_of(Integer)
+  end
```

Comment:
This duplicates the `provides some number for count` spec below.

Reply:
I don't mind that - I focus on what the specification specifies and I think it's ok if it specifies two different things using the same code to do it.